source/administration/upgrade-script.rst

```diff
@@ -0,0 +1,23 @@
+Upgrading Mattermost Server with a script
+=========================================
+
```

Comment:
Add the following here:

.. important:: This unofficial guide is maintained by the Mattermost community and this deployment configuration is not yet officially supported by Mattermost, Inc. Community testing, feedback, and improvements are welcome and greatly appreciated.